services/app-api/src/emailer/emails/newPackageStateEmail.ts

```diff
@@ -26,6 +26,7 @@ export const newPackageStateEmail = async (
     const receiverEmails = pruneDuplicateEmails([
         ...stateContactEmails,
         ...submitterEmails,
+        ...config.cmsReviewSharedEmails,
```

Comment:
Please add this line to all state emails (unlock, resubmit state emails as well)